src/template/runner.rs

```diff
@@ -64,8 +62,6 @@ fn bench<I: Clone, T>(func: impl Fn(I) -> T, input: I, base_time: &Duration) ->
     let mut timers: Vec<Duration> = vec![];
 
     for _ in 0..bench_iterations {
-        // need a clone here to make the borrow checker happy.
-        let cloned = input.clone();
         let timer = Instant::now();
         black_box(func(black_box(cloned)));
```

Comment:
```suggestion
        black_box(func(black_box(input)));
```
You've left in a reference to the removed clone.

Reply:
Thanks for catching it :)

Reply:
No worries. I've since been skimming through your solution repo and am learning a ton of rust tricks, so it's more than paid off!